src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SPLITTER = ",";
+    private static final String SUPPLY = "supply";
+    private static final int STEP_COUNT = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        writeToFile(toFileName, collectStatistics(readFromFile(fromFileName)));
```

Comment:
It isn't best practice to call methods in this way. 
Please call methods line by line

readFromFile(fromFileName)
collectStatistics 
writeToFile

Reply:
Agree, i hope its ok to stop at 2 lines